Desktop/mainwindow.cpp

```diff
@@ -1646,6 +1662,14 @@ void MainWindow::reportHere(QString dir)
 	_reporter = new Reporter(this, dir);
 }
 
+void MainWindow::generateWrappers(QString dir)
+{
+	connect(_engineSync, &EngineSync::moduleInstallationSucceeded, this, &MainWindow::generateWrappersForModule, Qt::QueuedConnection);
```

Comment:
`generateWrappers` should be renamed `initWrapperGeneration` or something. Because you arent actually generating anything here.